File: src/buldringno/wwwroot/app/core/domain/problem.ts
export class Problem {
    Id: number;
    Title: string;
    Uri: string;
    BoulderId: number;
    BoulderTitle: string;
    Grade: string;
    DateUploaded: Date;

    constructor(id: number,
        title: string,
        uri: string,
        boulderId: number,
        boulderTitle: string,
        grade: string,
        dateUploaded: Date) {
        this.Id = id;
        this.Title = title;
        this.Uri = uri;
        this.BoulderId = boulderId;
        this.BoulderTitle = boulderTitle;
        this.Grade = grade;
        this.DateUploaded = dateUploaded;
    }
}

File: src/buldringno/Entities/Problem.cs
using System;

namespace BuldringNo.Entities
{
    public class Problem : IEntityBase
    {
        public int Id { get; set; }
        public string Title { get; set; }
        public string DescriptionMain { get; set; }
        public string DescriptionSecondary { get; set; }
        public string Uri { get; set; }
        public virtual Boulder Boulder { get; set; }
        public int BoulderId { get; set; }
        public string GradeStandingStart { get; set; }
        public string GradeSitStart { get; set; }
        public double Height { get; set; }
        public int NumberOfPads { get; set; }
        public int NumberOfStars { get; set; }
        public string FirstClimberStanding { get; set; }
        public string FirstClimberSitStart { get; set; }
        public DateTime FirstClimberStandingDate { get; set; }
        public DateTime FirstClimberSitStartDate { get; set; }
        public DateTime DateUploaded { get; set; }
    }
}
